src/main/java/org/elasticsearch/index/query/QueryBuilders.java

```diff
@@ -59,11 +59,11 @@ public static MatchQueryBuilder matchQuery(String name, Object text) {
     /**
      * Creates a common query for the provided field name and text.
      *
-     * @param name The field name.
+     * @param fieldName The field name.
      * @param text The query text (to be analyzed).
      */
-    public static CommonTermsQueryBuilder commonTermsQuery(String name, Object text) {
-        return new CommonTermsQueryBuilder(name, text);
+    public static CommonTermsQueryBuilder commonTermsQuery(String fieldName, String text) {
+        return new CommonTermsQueryBuilder(fieldName, text);
```

Comment:
along the same lines as the value.toString comment above, this looks fishy. Beyond that, it's a breaking change for the java api, we need to think about it, we should try and keep bw comp at this stage.